iphone/Maps/UI/PlacePage/PlacePageManager/MWMPlacePageManager.mm

```diff
@@ -180,6 +180,14 @@ - (void)addBookmark:(PlacePageData *)data {
   [data updateBookmarkStatus];
 }
 
+- (void)updateBookmark:(PlacePageData *)data color:(MWMBookmarkColor)color category:(MWMMarkGroupID)category {
+  MWMBookmarksManager * bookmarksManager = [MWMBookmarksManager sharedManager];
+  auto const & markId = data.bookmarkData.bookmarkId;
+  [bookmarksManager updateBookmark:markId setGroupId:category title:data.previewData.title color:color description:data.bookmarkData.bookmarkDescription];
```

Comment:
nit: these ids are integers, no need to make a reference to an integer, it is clearer to pass by value:
```suggestion
  [bookmarksManager updateBookmark:data.bookmarkData.bookmarkId setGroupId:category title:data.previewData.title color:color description:data.bookmarkData.bookmarkDescription];
```